07LessonSeven/homework/css/style.css

```diff
@@ -0,0 +1,35 @@
+* {
+  font-family:Arial;
+}
+header {
+  margin-left: 100px;
+}
+#nav {
+  background-color: black;
+  padding: 20px 100px 20px;
+}
+li {
+  display: inline;
+  color: white;
+}
+
+main {
+  width: 1000px;
+  margin: 0px 100px;
+  padding: 25px;
+
+}
+.hire {
+  color:white;
+  background-color: green;
+  padding:25px;
+  width:30%;
```

Comment:
try changing this to make the hire and look classes be 50% instead. this way, they'll be centered and take up the whole screen.